backend/app/util.py

```diff
@@ -26,10 +29,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-# FIXME: Idk how to return proper HTTPException's to the frontend
 def decode_jwt(encoded_jwt: str):
-    idinfo = id_token.verify_oauth2_token(
-        encoded_jwt, requests.Request(), get_settings().google_client_id
-    )
-
-    return GoogleAuth(**idinfo)
+    try:
+        idinfo = id_token.verify_oauth2_token(
+            encoded_jwt, requests.Request(), get_settings().google_client_id
+        )
+        return GoogleAuth(**idinfo)
+
+    # Error for malformed jwt's
+    except MalformedError:
+        raise HTTPException(status_code=498, detail="Could not parse JWT")
```

Comment:
Could the error be helpful for debugging? 
```suggestion
    except MalformedError as e:
        raise HTTPException(status_code=498, detail=f"Could not parse JWT: {e}")
```

Reply:
The reason this is not there is because it gives specific information about what part of the JWT that it cannot parse. This might be a bit too much information?

Reply:
Hmm, that's true. Maybe we should use the environment

`from app.config import Environment, get_settings`
```suggestion
    except MalformedError as e:
        if get_settings().app_environment == Environment.DEVELOPMENT:
            raise HTTPException(status_code=498, detail=f"Could not parse JWT: {e}")
        raise HTTPException(status_code=498, detail="Could not parse JWT")
```

Reply:
Lets add it if it ever becomes an issue, but right now i cannot see how getting a part of a JWT that fails might help us 🤷🏽  I might be wrong tho